src/client/method-status.c

```diff
@@ -188,12 +188,15 @@ static void print_info_header(size_t name_col_width) {
         fprintf(stdout, "----------------\n");
 }
 
-#define PRINT_AND_ALIGN(x)                                       \
-        do {                                                     \
-                fprintf(stdout, "| %s\t", unit_info->x);         \
-                if (!unit_info->x || strlen(unit_info->x) < 6) { \
-                        fprintf(stdout, "\t");                   \
-                }                                                \
+#define PRINT_AND_ALIGN(x)                               \
+        do {                                             \
+                fprintf(stdout, "| %s\t", unit_info->x); \
+                if (unit_info->x) {                      \
```

Comment:
Previously, we printed a `\t` when the field x is not defined/empty or its length was smaller than 6 chars. This changes the `or` to an `and`. Is it not a valid case that a field is empty, so we need to tab here?